hack/istio/tempo/install-tempo-env.sh

```diff
@@ -0,0 +1,160 @@
+  -t|--tempo-ns:
+       Tempo namespace. Tempo by default.
+  -h|--help:
+       this message
+HELPMSG
+      exit 1
+      ;;
+    *)
+      echo "ERROR: Unknown argument [$key]. Aborting."
+      exit 1
+      ;;
+  esac
+done
+
+SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+if [ "${DELETE_ALL}" == "true" ]; then
+  DELETE_TEMPO="true"
+fi
+
+if [ "${DELETE_TEMPO}" == "true" ]; then
+  echo -e "Deleting tempo \n"
+  ${CLIENT_EXE} delete -f https://github.com/cert-manager/cert-manager/releases/latest/download/cert-manager.yaml
+  ${CLIENT_EXE} delete -f https://github.com/grafana/tempo-operator/releases/latest/download/tempo-operator.yaml
+  ${CLIENT_EXE} delete secret -n ${TEMPO_NS} tempostack-dev-minio
+  ${CLIENT_EXE} delete TempoStack smm -n ${TEMPO_NS}
+  ${CLIENT_EXE} delete ns ${TEMPO_NS}
+  if [ "${DELETE_ALL}" == "true" ]; then
+    ${SCRIPT_DIR}/../install-istio-via-istioctl.sh -c ${CLIENT_EXE} -di true
+    ${SCRIPT_DIR}/../install-bookinfo-demo.sh -c ${CLIENT_EXE} -db true
+  fi
+else
+  echo -e "Installing cert manager...\n"
+  ${CLIENT_EXE} apply -f https://github.com/cert-manager/cert-manager/releases/latest/download/cert-manager.yaml
+  echo -e "Waiting for cert-manager pods to be ready... \n"
+  $CLIENT_EXE wait pods --all -n cert-manager --for=condition=Ready --timeout=5m
+
+  echo -e "Installing latest Tempo operator \n"
+  ${CLIENT_EXE} apply -f https://github.com/grafana/tempo-operator/releases/latest/download/tempo-operator.yaml
+  echo -e "Waiting for Tempo operator to be ready... \n"
+  $CLIENT_EXE wait pods --all -n tempo-operator-system --for=condition=Ready --timeout=5m
+
+  ${CLIENT_EXE} create namespace ${TEMPO_NS}
+
+  echo -e "Installing minio and create secret \n"
+  ${CLIENT_EXE} apply --namespace ${TEMPO_NS} -f ${SCRIPT_DIR}/minio.yaml
+
+  ${CLIENT_EXE} create secret generic -n ${TEMPO_NS} tempostack-dev-minio \
+    --from-literal=bucket="tempo-data" \
+    --from-literal=endpoint="http://minio:9000" \
+    --from-literal=access_key_id="minio" \
+    --from-literal=access_key_secret="minio123"
+
+  echo -e "Installing tempo \n"
+  ${CLIENT_EXE} apply -n ${TEMPO_NS} -f - <<EOF
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: smm
+spec:
+  storageSize: 1Gi
+  storage:
+    secret:
+      type: s3
+      name: tempostack-dev-minio
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: false
+EOF
+
+
+  echo "${SCRIPT_DIR}"
+
+  if [ "${INSTALL_ISTIO}" == "true" ]; then
+    echo -e "Installing istio \n"
+    ${SCRIPT_DIR}/../install-istio-via-istioctl.sh -c ${CLIENT_EXE} -a "prometheus grafana" -s values.meshConfig.defaultConfig.tracing.zipkin.address="tempo-smm-distributor.tempo:9411"
+  fi
+
+  if [ "${INSTALL_ISTIO}" == "true" ]; then
+    OUTPUT_DIR="${OUTPUT_DIR:-${SCRIPT_DIR}/../../../_output}"
+    ISTIO_DIR=$(ls -dt1 ${OUTPUT_DIR}/istio-* | head -n1)
+    echo ${ISTIO_DIR}
```

Comment:
Again, just to make it clear what this is to the user debugging, add a label... otherwise, the person is just going to see some random file path spit out in the logs without any context explaining what it is:
```suggestion
    echo "Istio directory where the Kiali addon yaml should be found: ${ISTIO_DIR}"
```

Reply:
Yes, right, it was for debugging, my intention was to delete it but not sure if it might be helpful?

Reply:
They are hack scripts  - can't have too much debug log output :) This can be helpful in cases where you have multiple istio directories downloaded and you want to make sure you are getting the correct one (I do this all the time - download multiple istio versions).